phpBB/language/en/acp/posting.php

```diff
@@ -78,13 +78,13 @@
 	'TOO_MANY_BBCODES'		=> 'You cannot create any more BBCodes. Please remove one or more BBCodes then try again.',
 
 	'tokens'	=>	array(
-		'TEXT'			=> 'Any text, including foreign characters, numbers, etc… You should not use this token in HTML tags. Instead try to use IDENTIFIER, INTTEXT or SIMPLETEXT.',
```

Comment:
Does that means is possible to use {TEXT} token inside a HTML attribute now?

Reply:
It's always been possible and as far as I went back in the repository's history, it's always been safe to use. I think the warning dates back to a very old version of phpBB.

Reply:
Thx, that was my impression as well.

Reply:
There are though check-in and language strings still like `BBCODE_DANGER` and the logic in `includes/acp/acp_bbcodes.php`

```
				else
				{
					confirm_box(false, $user->lang['BBCODE_DANGER'], build_hidden_fields(array_merge($hidden_fields, array(
						'action'				=> $action,
						'bbcode'				=> $bbcode_id,
						'bbcode_match'			=> $bbcode_match,
						'bbcode_tpl'			=> htmlspecialchars($bbcode_tpl),
						'bbcode_helpline'		=> $bbcode_helpline,
						'display_on_posting'	=> $display_on_posting,
						)))
					, 'confirm_bbcode.html');
				}
```